mdn/tests/test_tasks.py

```diff
@@ -99,11 +99,11 @@ def tearDown(self):
 
     def test_good_call(self):
         data = {
-            'locale': 'en-US', 'title': 'display',
-            'url': '/en-US/docs/Web/CSS/display',
+            'locale': 'en-US', 'title': 'float',
+            'url': '/en-US/docs/Web/CSS/float',
             'translations': [{
-                'locale': 'es', 'title': 'display',
-                'url': '/es/docs/Web/CSS/display'}]}
+                'locale': 'es', 'title': 'float',
+                'url': '/es/docs/Web/CSS/float'}]}
```

Comment:
The test setup creates a feature based on Web/CSS/float, and then the meta was returning Web/CSS/display - it doesn't really matter for the test (downloading the URL is a follow-on task), but I wanted to make it look more consistent.

And then I made the same mistake in the redirect_to_zone test.  Really should have waited until morning to submit :(